### ISSUE TITLE
Outbound Messages

### ISSUE THREAD

**Issue by ricco-douro** [canove/whaticket#234]
Hello.

I have created a new contact, but i am unable to initiate a conversation. 

Only after the contact exchanges a message with me, can I then open a new ticket (even though I truncated all contacts, messages and tickets tables, erasing all traces of the exchange)

Is this a feature, a bug, or is there by design?
